playbooks/roles/ocim/tasks/vagrant.yml

```diff
@@ -10,6 +10,11 @@
     force: no
   become_user: "{{ www_user }}"
 
+- name: Install javascript dependencies
+  shell: DEBIAN_FRONTEND=noninteractive make install_js_dependencies
+  args:
+    chdir: "{{ opencraft_root_dir }}"
```

Comment:
@viadanna, it is better to explicitly run this as the `www_user` like it is done while installing the python requirements [here](https://github.com/open-craft/ansible-playbooks/blob/master/playbooks/roles/ocim/tasks/common.yml#L13).